src/mapeo-manager.js

```diff
@@ -78,23 +80,34 @@ export class MapeoManager extends TypedEmitter {
    * @param {Object} opts
    * @param {Buffer} opts.rootKey 16-bytes of random data that uniquely identify the device, used to derive a 32-byte master key, which is used to derive all the keypairs used for Mapeo
    * @param {string} opts.dbFolder Folder for sqlite Dbs. Folder must exist. Use ':memory:' to store everything in-memory
+   * @param {string} [opts.projectMigrationFolder] path for drizzle migration folder for project
+   * @param {string} [opts.clientMigrationFolder] path for drizzle migration folder for client
    * @param {string | import('./types.js').CoreStorage} opts.coreStorage Folder for hypercore storage or a function that returns a RandomAccessStorage instance
    * @param {{ port?: number, logger: import('fastify').FastifyServerOptions['logger'] }} [opts.mediaServerOpts]
    */
-  constructor({ rootKey, dbFolder, coreStorage, mediaServerOpts }) {
+  constructor({
+    rootKey,
+    dbFolder,
+    projectMigrationFolder = '../drizzle/project',
+    clientMigrationFolder = '../drizzle/client',
```

Comment:
Shouldn't use a default value here

```suggestion
    projectMigrationsFolder,
    clientMigrationsFolder,
```